.github/workflows/monthly_release.yml

```diff
@@ -41,3 +41,5 @@ jobs:
 
             [1]: https://github.com/peter-evans/create-pull-request
           branch: new-release
+          base: master
+          delete-branch: true
```

Comment:
qq: what does delete branch do?

Reply:
Deletes the branch new-release in this case. Right now I am looking into weather the python-semantic-release action requires the branch to already be exist or if it can checkout a ew branch.